### PR TITLE
Plot global plasma parameters vs time

### PR DESCRIPTION
**Pull request by anteplu**
This PR adds a few more plasma parameters to be calculated by the post-processing script and makes a new plot config to plot them vs time. 
New parameters in post-processing:
- volume average electron and main ion temperatures
- volume average electron and main ion densities
- q95 (+ a function to calculate q95)

It would be nice to have an option to remove a time slider for plots that show parameters only vs time.
Issue open: https://github.com/google-deepmind/torax/issues/616

Example of the plot output with `torax/plotting/configs/global_params_plot_config.py`:
Figure_2_gl_plt

### CODE REVIEW

File: torax/plotting/configs/global_params_plot_config.py [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+PLOT_CONFIG = plotruns_lib.FigureProperties(
+    rows=2,
+    cols=3,
+    tick_fontsize=8,
+    axes_fontsize=8,
+    default_legend_fontsize=7,
+    figure_size_factor=5,
+    title_fontsize=12,
+    axes=(
+        # For chi, set histogram percentile for y-axis upper limit, due to
```
Comment (jcitrin) [inline torax/plotting/configs/global_params_plot_config.py:28]: This comment can be removed

File: torax/plotting/configs/global_params_plot_config.py [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+"""Plotting configuration for global plasma parameters vs time."""
+
+from torax.plotting import plotruns_lib
+
+PLOT_CONFIG = plotruns_lib.FigureProperties(
+    rows=2,
+    cols=3,
+    tick_fontsize=8,
+    axes_fontsize=8,
+    default_legend_fontsize=7,
```
Comment (jcitrin) [inline torax/plotting/configs/global_params_plot_config.py:24]: looking at the example plot pasted in the PR, it feels like either all fontsizes can be increased or the figure_size_factor reduced. Should experiment with that a bit more and find more optimal values for clear plots

**Review by jcitrin — CHANGES_REQUESTED** [review]
Looks good and great to have your first PR in :)

After the minor changes requested and signing the CLA, we'll bring this in.

File: torax/plotting/configs/global_params_plot_config.py (hunk 1 above)
Comment (anteplu) [reply torax/plotting/configs/global_params_plot_config.py]: Done [4929853](https://github.com/google-deepmind/torax/pull/617/commits/4929853bac3e43317d88aea63c0fd326787d51dd).

File: torax/plotting/configs/global_params_plot_config.py (hunk 2 above)
Comment (anteplu) [reply torax/plotting/configs/global_params_plot_config.py]: Done [4929853](https://github.com/google-deepmind/torax/pull/617/commits/4929853bac3e43317d88aea63c0fd326787d51dd).
The example plot was updated too.

**Comment by jcitrin** [comment]
Looks great, thanks, will bring it in

**Review by jcitrin — APPROVED** [review]

**Review by jcitrin — COMMENTED** [review]
A few minor changes have been done on our end before merging, noticed by the linter, and will be visible in the merge commit. 

No action needed from you, but mentioning for visibility.

1. The tuple typehint is not necessary for a single output in the q95 function. Could just be array_typing.ScalarFloat. 
2. There were a couple of stray comments removed, related to IPython.embed
3. Various formatting issues raised by the linter. I recommended installing and embedding pylint within your VScode to catch these ahead of time.

**Comment by jcitrin** [comment]
Merged with https://github.com/google-deepmind/torax/commit/2af8b6762991a419dec1877151c3d9398cd462b1

**Pull request merged**